app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +17,11 @@
         tools:ignore="QueryAllPackagesPermission" />
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
-    <uses-permission android:name="android.permission.CAMERA" android:required="false" />
+    <uses-permission android:name="android.permission.CAMERA"/>
```

Comment:
I've removed the `optional` attribute as I don't see it in the docs https://developer.android.com/guide/topics/manifest/uses-permission-element